### ISSUE TITLE
Certain combinations cause Watch Page Issues

### ISSUE THREAD

**Issue by IdrisQe** [lightbeam24/CustomTube#195]
2020 Layout
Player Style: Auto
Watch Version: Current

-Autoplay button is missing.
-Video title, channel name, video description, etc. is missing.

![image](https://github.com/lightbeam24/CustomTube/assets/40530630/33a0112e-b6f7-41e3-b8cc-08ee5da770be)

2020 Layout
Player Style: Auto
Show Channel Videos Link: Off
Watch Version: Watch9B

-Description uses black text even when using Youtube's Dark Mode, making it nearly illegible.
-Channel Videos Link still shows despite being disabled.

![image](https://github.com/lightbeam24/CustomTube/assets/40530630/3d33c69a-b258-4017-b7f3-fd6a342d0540)

**Comment by lightbeam24** [comment]
> Watch Version: Current

Yes, I'm aware that this option is completely broken in the current 1.3.0 Beta, as is the "Disabled" layout option.

> Show Channel Videos Link: Off

This option is not functional in 1.3.0 yet. This isn't the only one either.

> -Description uses black text even when using Youtube's Dark Mode, making it nearly illegible.

Dark mode optimization has not been done yet on this new HTML.

All of these will hopefully be fixed for the full 1.3.0 release.
